integrations/parsely.php

```diff
@@ -41,15 +41,18 @@ public function load(): void {
 	 * @return array
 	 */
 	public function wp_parsely_credentials_callback( $original_credentials ) {
-		$config = $this->get_config();
+		$config      = $this->get_config();
+		$credentials = array();
 
-		if ( empty( $config ) ) {
-			return $original_credentials;
-		}
-
-		return array(
+		// If config provided by VIP is empty then take original credentials else take
+		// credentials from config.
+		$credentials = empty( $config ) ? $original_credentials : array(
```

Comment:
Small suggestion - may be better to check for the keys you need explicitly (`site_id` and `api_secret`) rather than any `$config` value. If `$config` currently or in the future accepts other configuration keys, this code would set both values to `null` if any other configuration key is set.

```suggestion
		$credentials = ( isset( $config[ 'site_id' ] ) && isset( $config[ 'api_secret' ] ) ? array( ... ) : $original_credentials;
```

Reply:
`$original_credentials` means credentials that we got from plugin so I think its fine to return as it is without any extra checks to keep things simple. Later we can improve if needed.